spring-kafka-docs/src/main/antora/modules/ROOT/pages/micrometer.adoc

```diff
@@ -0,0 +1,60 @@
+= Micrometer Support
+
+Spring for Apache Kafka provides support for Micrometer.
+
+[[listener-obs]]
+== Listener Observations
+
+When you have `micrometer-observation` on the class path, and you configure a
+`ObservationRegistry` in the application context, the container will automatically create
+`Observation`s for consumed records.
```

Comment:
Our preferences is `One sentence per line`: https://asciidoctor.org/docs/asciidoc-recommended-practices/